mu: add page (#7572)
tldr-pages/tldr: pages/common/mu.md

# mu

> Index and search emails from a local Maildir.

> > More information: <https://www.djcbsoftware.nl/code/mu/>.

Reviewer: Although the given link is from official page, but the page only contain a link of a cheat sheet rather than detailed manual. I'd prefer to use the man page from https://manned.org/mu.
```suggestion
> More information: <https://manned.org/mu>.
```

Author: You're right, apart from fairly hidden details about `mu init` which are not in the man page, the manual there is only about mu4e.

Author: Actually, this man page was last updated in 2013, so before the 0.9.6 version introducing `mu init` and some other changes. If it's fine to link to any online man page, https://man.cx/mu(1) seems to have been updated this month. I should have checked that before committing.

- Initialize the email database, optionally specifying the Maildir directory and email addresses belonging to self :

`mu init --maildir={{/path/to/directory}} --my-address={{name@example.com}}`

- Index new emails:

`mu index`

- Find messages with a keyword (in message body, subject, sender, ...)
`mu find {{keyword}}`

- Find messages to Alice with subject `jellyfish` containing the words `apples` or `oranges`:

`mu find to:{{alice}} subject:{{jellyfish}} {{apples}} OR {{oranges}}`

> - Find unread messages about words starting with `soc` (the `*` only works at the end of the search term) in the Sent Items folder:
> `mu find 'subject:{{soc}}*' flag:{{unread}} maildir:'{{/Sent Items}}'`

Reviewer: If the syntax of maildir is relative directory, it should be `maildir:'{{path/to/Sent Items}}'`.
If the syntax is / + folder name, it should be `maildir:'/{{Sent Items}}'`.

```suggestion
- Find unread messages about words starting with `soc` (the `*` only works at the end of the search term) in the `Sent Items` folder:

`mu find 'subject:{{soc}}*' flag:{{unread}} maildir:'{{/Sent Items}}'`
```

# mu

> Index and search emails from a local Maildir.

> > More information: <https://man.cx/mu(1)>.

Reviewer 2: ```suggestion
> More information: <https://man.cx/mu>.
```

> - Initialize the email database, optionally specifying the Maildir directory and email addresses belonging to self:

Reviewer 2: We try to avoid `your`, `you`, etc.
```suggestion
- Initialize the email database, optionally specifying the Maildir directory and email addresses:
```

> `mu init --maildir={{/path/to/directory}} --my-address={{name@example.com}}`

Reviewer 2: ```suggestion
`mu init --maildir={{path/to/directory}} --my-address={{name@example.com}}`
```

- Index new emails:

`mu index`

> - Find messages with a keyword (in message body, subject, sender, ...):

Reviewer 2: ```suggestion
- Find messages using a specific keyword (in message body, subject, sender, ...):
```

`mu find {{keyword}}`

- Find messages to Alice with subject `jellyfish` containing the words `apples` or `oranges`:

`mu find to:{{alice}} subject:{{jellyfish}} {{apples}} OR {{oranges}}`

- Find unread messages about words starting with `soc` (the `*` only works at the end of the search term) in the Sent Items folder:

`mu find 'subject:{{soc}}*' flag:{{unread}} maildir:'/{{Sent Items}}'`

> - Find messages from Sam with attached images, between 2kB and 2Mb, written in 2021:

Reviewer 2: Is this MiB? 1024 or 1000?
```suggestion
- Find messages from Sam with attached images, between 2 KiB and 2 MiB, written in 2021:
```

Reviewer 3: Great catch. `Mb` can commonly mean a Mega*bit* rather than a Mega*byte*, so this could be a typo.
